### PR TITLE
Fix defaults for mt_rand

### PR DESCRIPTION
**Pull request by antoniopaisfernandes**

### CODE REVIEW

**Comment by jdferreira** [comment]
I was about to submit this exact same PR, along with the better description and all! Thanks :)

**Comment by antoniopaisfernandes** [comment]
:) glad to help!

File: standard/standard_2.php [hunk 1]
```diff
@@ -993,7 +993,7 @@ function srand ($seed = null, $mode = MT_RAND_MT19937) {}
 function getrandmax () {}
 
 /**
- * Generate a better random value
+ * Generate a random value via the Mersenne Twister Random Number Generator
```
Comment (andrey-sokolov) [inline standard/standard_2.php]: Please, add description for parameters

Comment (andrey-sokolov) [reply standard/standard_2.php]: In this PR we are going to add default values, but according to description `max` default value is  `mt_getrandmax()` so it'll be better to mention it in docs

Comment (antoniopaisfernandes) [reply standard/standard_2.php]: Done

**Comment by Virakal** [comment]
Just checking - shouldn't these be `@param int $min` and `@param int max` with type definitions?

**Comment by antoniopaisfernandes** [comment]
Nice catch! Thanks

**Pull request merged**